### PR TITLE
Add Date Validator and tests

### PR DESCRIPTION
**Pull request by bulat-gab**

### CODE REVIEW

**Comment by CraigBanach** [comment]
If you add a line into the list of Validators in the Readme noting your new DateValidator, I think this would be useful.

See: (https://github.com/afucher/Inquirer/pull/18/commits/bc2854cac22ca74a28473707ab552c51c6f0c026)

**Comment by bulat-gab** [comment]
> If you add a line into the list of Validators in the Readme noting your new DateValidator, I think this would be useful.
> 
> See: ([bc2854c](https://github.com/afucher/Inquirer/commit/bc2854cac22ca74a28473707ab552c51c6f0c026))

Done

File: test/InquirerUnitTest/Validators/DateValidatorUnitTest.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        }
+
+        [Theory]
+        [InlineData("2020-13-01")]
+        [InlineData("-0020-12-01")]
+        [InlineData("2020-12-32")]
+        [InlineData("invalid")]
+        [InlineData("")]
+        [InlineData(null)]
+        public void SShouldNotAcceptNonDateValues(string aNonDateValue)
```
Comment (afucher) [inline test/InquirerUnitTest/Validators/DateValidatorUnitTest.cs]: Just a little typo here, it has an extra `S`

**Review by afucher — CHANGES_REQUESTED** [review]
Thanks for your PR!

File: test/InquirerUnitTest/Validators/DateValidatorUnitTest.cs (hunk 1 above)
Comment (bulat-gab) [reply test/InquirerUnitTest/Validators/DateValidatorUnitTest.cs]: Done

**Comment by afucher** [comment]
@bulat-gab your PR has some conflicts, could you please rebase it from master?

**Comment by bulat-gab** [comment]
> @bulat-gab your PR has some conflicts, could you please rebase it from master?

Done

**Review by afucher — APPROVED** [review]
Thanks for your contribution! 🎉

**Pull request merged**